workflow/rules/process/exclude-countries.smk

```diff
@@ -0,0 +1,15 @@
+"""Rule to document countries to be excuded (no .tif file)
+
+"""
```

Comment:
I wonder if this rule could be handled transparently by snakemake itself. We can think about it when considering the whole workflow and how snakemake can best be used to manage the various combinations of boxes and hazards.

Reply:
Happy to hear thoughts on this